core/src/main/java/org/pentaho/reporting/platform/plugin/PentahoReportEnvironment.java

```diff
@@ -63,6 +64,15 @@ public PentahoReportEnvironment( final Configuration configuration, final String
     setLocale( LocaleHelper.getLocale() );
   }
 
+  private static String getSessionTenantId() {
+    IPentahoSession session = PentahoSessionHolder.getSession();
+    if ( session == null ) {
+      return JcrTenantUtils.getDefaultTenantPath();
+    }
+
+    return JcrTenantUtils.getTenant( session.getName(), true ).getId();
```

Comment:
**no action needed, not a must change, just a note for future reference**

Since you aren't using the value from getSession(), #getSessionTenantId() could be converted to a one line return statement  and/or one-line statement could just be replaced function call in PentahoReportEnvironment#getEnvironmentProperty( String )
```
return PentahoSessionHolder.getSession() == null 
  ? JcrTenantUtils.getDefaultTenantPath();
  : JcrTenantUtils.getTenant( session.getName(), true ).getId();
```
  :

Reply:
Note `session` is being used in the last line as well.